content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -23,7 +23,7 @@ By default, Istio defines and generates a set of standard metrics (e.g.
 
 Istio uses the Envoy proxy to generate metrics and provides its configuration in
 the `EnvoyFilter` at
-[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.7.yaml).
+[`manifests/charts/istio-control/istio-discovery/templates`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_{{< istio_version >}}.yaml).
```

Comment:
My only concern is we have removed the version from the link and we are now pointing to the folder. I suspect that's because we single quotes. Might try removing the quotes.

Reply:
Removing quotes will render it as plain text and it won't be more recognizable. We are saying in the sentence that `EnvoyFilter is at location` so it should be fine to point them to the directory. Anyways we are redirecting users to the exact file they are looking for the configuration.

Reply:
It would be plain text, but it is still highlighted in blue as a link which should be OK as it is a link.

Reply:
Yup, it looks good without a qoute. Thanks.